src/odemis/util/no_conflict.py

```diff
@@ -13,7 +14,10 @@ def skip_redundant(iterable, skipset=None):
 
 
 def remove_redundant(metaclasses):
-    skipset = set([types.ClassType])
+    if platform.python_version()[0] == 3:
```

Comment:
Please use the following to check for Python 3 (apparently it's a tiny bit better):
`sys.version_info[0] >= 3`